app/config.py

```diff
@@ -69,6 +72,9 @@ class DevelopmentConfig:
     PORT = env.int("PORT", default="9092")
     LOG_LEVEL = env("LOG_LEVEL", default="INFO")
 
+    DOMAIN_URL_EN = env.str("DOMAIN_URL_EN", default="http://localhost:9092")
+    DOMAIN_URL_CY = env.str("DOMAIN_URL_CY", default="http://localhost:9092")
```

Comment:
You might want to make these default to the host and port values above: `default=f"http://{HOST}:{PORT}"`

Reply:
And the same for anything that happens to match the `PORT` or `HOST` below.

Reply:
I did think about this - I think I'm happy as it's consistent with what was already there